File: Semana 4/TU_5/src/main/java/com/mycompany/tu_5/Persona.java
package com.mycompany.tu_5;

public class Persona {
    private String nombre;
    private int edad;

    public Persona(String pNombre, int pEdad) {
        this.nombre = pNombre;
        this.edad = pEdad;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String pNombre) {
        this.nombre = pNombre;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int pEdad) {
        this.edad = pEdad;
    }

    @Override
    public String toString() {
        return "Persona{" + "nombre=" + nombre + ", edad=" + edad + '}';
    }
    

   
}

File: Semana 6/TU_6/TU_6/Persona.cpp
#include "Persona.h"

Persona::Persona(){
	this->edad = 0;
	this->nombre = "indef";
}

Persona::Persona(int pEdad, string pNombre){
	this->edad = pEdad;
	this->nombre = pNombre;
}

void Persona::setNombre(string pNombre){
	this->nombre = pNombre;
}

void Persona::setEdad(int pEdad){
	this->edad = pEdad;
}

string Persona::getNombre(){
	return this->nombre;
}

int Persona::getEdad() const{
	return this->edad;
}

Persona::~Persona()
{
}
